docs/k0s-in-docker.md

```diff
@@ -19,7 +19,7 @@ token=$(docker exec -t -i k0s k0s token create --role=worker)
 
 Then join a new worker by running the container with:
 ```
-docker run -d --name k0s-worker1 --hostname k0s-worker1 --privileged -v /var/lib/k0s docker.pkg.github.com/k0sproject/k0s/k0s:<version> k0s worker $token
+docker run -d --name k0s-worker1 --hostname k0s-worker1 --privileged -v /var/lib/k0s docker.io/k0sproject/k0s:latest k0s worker $token
```

Comment:
I think, it's better to have both images as an example here, because as far as I remember docker.io registry could have limits now and if people would run this example while we hit limits there could be questions and incoming issues

Reply:
This change was about aligning github/docker hub links. Previously 2/3 links were already updated, but this one forgotten. However, I could add a separate note about this github link as an alternative to docker hub.